### DISCUSSION TITLE
Addon stuck when I attempt to import/link model

### DISCUSSION THREAD

**Discussion by Imyamoe** [carson-katri/dream-textures#782] [General]
I'm using the addon version 0.3.1 on Blender 4.1. And when I'm trying to import base SDXL there are no errors, but the importing bar is stuck at 0%. Tried to manually select version, but the result is same. Also when I'm trying to link the model, the Blender stuck with no response, and the only thing I can do is to force close it from the Windows tasks. Tried to leave it loading like that for the night, but it seems hanging solid.

UPD: And it is same for downloading a default model in preferences too. The addon does not attempt to download it, and stays at 0%

Anyone had this problem too, or have any ideas on how can I fix that?

**Comment by SatanBroccoli** [comment]
I have the same problem

**Comment by bananaman1983** [comment]
same here no clue

**Comment by Imyamoe** [comment]
It is working on 3.6 though
